test/TesterInternal/OrleansRuntime/Streams/FixedSizeBufferTests.cs

```diff
@@ -35,7 +35,7 @@ public void EmptyBlockGetSegmentTooLargeBvt()
             Assert.False(buffer.TryGetSegment(TestBlockSize + 1, out segment), "Should not be able to get segement that is bigger than block.");
            Assert.Null(segment.Array);
             Assert.Equal(0, segment.Offset);
-            Assert.Equal(0, segment.Count);
+            Assert.Empty(segment);
```

Comment:
Build Failure: Assert.Emtpy would enumerate, and it is not supported if the underlying array is empty. Please revert to previous assertion. Same with FillBlockTestBvt